packages/elements-core/src/components/TryIt/build-request.ts

```diff
@@ -300,3 +314,17 @@ function uriExpand(uri: string, data: Dictionary<string, string>) {
     return data[value] || value;
   });
 }
+
+function getAcceptedMimeTypes(httpOperation: IHttpOperation): string[] {
+  return [
+    ...new Set(
+      httpOperation.responses
+        .map(response =>
+          response === undefined || response.contents === undefined
+            ? []
+            : response.contents.map(contentType => contentType.mediaType),
+        )
+        .reduce((a, b) => [...a, ...b], []),
+    ),
+  ];
+}
```

Comment:
```suggestion

export function getAcceptedMimeTypes(httpOperation: IHttpOperation): string[] {
  return Array.from(
    new Set(
      httpOperation.responses.flatMap(response =>
        response === undefined || response.contents === undefined
          ? []
          : response.contents.map(contentType => contentType.mediaType),
      ),
    ),
  );
}
```

Reply:
I think the Array.from and flatMap are more readable - and flatMap will likely perform better than the map / reduce combo, although that might not matter given the limited number of responses in any given operation.